deepchem/models/openvino_model.py

```diff
@@ -0,0 +1,86 @@
+import os
+import subprocess
+import numpy as np
+import tensorflow as tf
+from tensorflow.python.framework.convert_to_constants import convert_variables_to_constants_v2
+
+try:
+  import mo_tf
+  from openvino.inference_engine import IECore
+  is_available = True
+except:
+  is_available = False
+  pass
+
+
+class OpenVINOModel:
```

Comment:
Could you add a docstring here with documentation on OpenVINO and some more information about the class?

Reply:
Done